Remotion/Data/DomainObjects.Validation.IntegrationTests/Testdomain/StubStorageProvider.cs

```diff
@@ -105,6 +109,12 @@ public override ObjectID CreateNewObjectID (ClassDefinition classDefinition)
       return new ObjectID(classDefinition.ID, Guid.NewGuid());
     }
```

Comment:
With integration tests outside of the core domain, I don't think we need the CheckClassDefinition call.